protostar-cli/src/main.rs

```diff
@@ -94,6 +97,23 @@ async fn main() -> Result<()> {
             // dbg!(declare);
             Ok(())
         }
+        Some(Commands::Call(args)) => {
+            let block_id = get_block_id(&args.block_id).unwrap();
+
+            let result = starknet_commands::call::call(
+                &args.contract_address,
+                &args.func_name,
+                &args.calldata,
+                &provider,
+                &block_id,
+            )
+            .await?;
+
+            // todo (#2107): Normalize outputs in CLI
+            eprintln!("Call response: {:?}", result);
```

Comment:
Why are we printing to stderr? If it's for debugging you can use `dbg!` macro though it prints to stdout I believe.

Reply:
I've consulted it with @THenry14 and we can leave it as is for now. There will be bigger issue to address all commands and their prints

Reply:
https://github.com/software-mansion/protostar/issues/2107